src/main/resources/templates/email/NewInterview_Interviewee.txt

```diff
@@ -0,0 +1,18 @@
+Hello {{interviewee_first_name}},
+
+You have been registered for an interview on {{formatted_date}}. You are the interviewee. You will be be mock interviewed by {{interviewer_first_name}} for the position of: {{position}}.
```

Comment:
Duplicate "be", probably remove "mock" from where it is now and say "you have been registered for a mock interview on"